src/hdmf/backends/hdf5/h5tools.py

```diff
@@ -30,7 +30,7 @@
 class HDF5IO(HDMFIO):
 
     @docval({'name': 'path', 'type': str, 'doc': 'the path to the HDF5 file'},
-            {'name': 'manager', 'type': BuildManager, 'doc': 'the BuildManager to use for I/O', 'default': None},
+            {'name': 'manager', 'type': (TypeMap, BuildManager), 'doc': 'the BuildManager to use for I/O', 'default': None},
```

Comment:
The doc should be updated to reflect the new behavior. Should the name of the arg also change?

Reply:
The name should stay, for backwards compatibility.